pkg/controlplane/instance.go

```diff
@@ -125,6 +124,9 @@ const (
 	DefaultEndpointReconcilerInterval = 10 * time.Second
 	// DefaultEndpointReconcilerTTL is the default TTL timeout for the storage layer
 	DefaultEndpointReconcilerTTL = 15 * time.Second
+	// DefaultPeerEndpointReconcilerTTL is the default TTL timeout for peer endpoint
+	// leases on the storage layer
+	DefaultPeerEndpointReconcilerTTL = 15 * time.Second
```

Comment:
Have split them apart because in a generic world these are different things. Could be convinced to remove this change here, and move it to the peer endpoint commit later.